### ISSUE TITLE
urlbar visible after page load

### ISSUE THREAD

**Issue by paulrouget** [browserhtml/browserhtml#970]
<img width="767" alt="screen shot 2016-03-29 at 05 50 11" src="https://cloud.githubusercontent.com/assets/373579/14097656/93bac7c6-f573-11e5-80da-ea899d4c764e.png">

**Comment by paulrouget** [comment]
Only happens on a slow connection.

Apparently, it happens if I press enter before ddg get a change to returns its result back.

**Comment by paulrouget** [comment]
> gozala
> I think we focus input when suggestions arrive here
> and we do not really check if it’s visible or not

**Comment by Gozala** [comment]
Ok so I dogged into this & as I suspected issue is that when we get a query result we update with selected completion for the first suggestion. The problem is nothing really checks if input still visible or not to decide if suggestion is still still relevant or not.

We should probably check if input is still focused before updating selection here:
https://github.com/browserhtml/browserhtml/blob/master/src/browser/input.js#L112-L121

Or alternative abort queries once user navigates to page. Or both.

I'll submit a patch to fix this.
